### DISCUSSION TITLE
Creates KP when installed w/ AirportBrcmFixup, BrcmBluetoothInjector, BrcmFirmwareData, BrcmPatchRAM3

### DISCUSSION THREAD

**Discussion by justinwallis** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#22] [Q&A]
Can you possibly make a way to have this plugin/kexts working in conjunction with the kexts for internal wifi/BT; AirportBrcmFixup, BrcmBluetoothInjector, BrcmFirmwareData, BrcmPatchRAM3

It gives me KP's on boot when i have both installed (Dell e7450 Opencore 0.6.4) 

Build Link: https://osxlatitude.com/index.php?/topic/8514-dell-latitude-e7450-el-capitan-clover-uefi-only/&do=findComment&comment=54665

Thank you

**Comment by chris1111** [comment]
Of course if you have AirportBrcmFixup, don't use both
